docs/clients/cognito-idp.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: client
+category: clients
+name: CognitoIdentityProvider
+package: async-aws/cognito-identity-provider
+---
+
+## Usage
+
+
+### Create a user as an admin
+
+```php
+use AsyncAws\CognitoIdentityProvider\CognitoIdentityProviderClient;
+use AsyncAws\CognitoIdentityProvider\ValueObject\AttributeType;
+
+$cognitoIdp = new CognitoIdentityProviderClient();
+
+$cognitoIdp->adminCreateUser([
+    'UserPoolId' => 'us-east-1_1337oL33t',
+    'Username' => '1c202820-8eb5-11ea-bc55-0242ac130003',
+    'UserAttributes' => [
+        new AttributeType(['Name' => 'phone_number', 'Value' => '+33600000000'])
+    ],
+]);
+```
+
+### Get a user as an admin
+
+```php
+use AsyncAws\CognitoIdentityProvider\CognitoIdentityProviderClient;
+
+$cognitoIdp = new CognitoIdentityProviderClient();
+
+$result = $cognitoIdp->adminGetUser([
+    'UserPoolId' => 'us-east-1_1337oL33t',
+    'Username' => '1c202820-8eb5-11ea-bc55-0242ac130003',
+    'UserAttributes' => [
+        new AttributeType(['Name' => 'phone_number', 'Value' => '+33600000000'])
+    ],
+]]);
+
+echo 'User status is: ' . $result->getUserStatus() . PHP_EOL;
+$givenNameAttribute = array_filter(
+    $result->getUserAttributes(),
+    function ($attr) { return $attr->getName() === 'given_name'); } ,
```

Comment:
Let's use PHP 7.2 code even in examples.

Reply:
seems valid PHP 7.2 isn't it ?

Reply:
Oops. Sorry. I was sure it said fn($attr)

Reply:
Sorry yes I did it yesterday but fixed since :wink: